README.md

```diff
@@ -1,42 +1,46 @@
-# Clientside InstantSearch Demo
-## Introduction
+# Clientside Instantsearch Demo
 
-This Instantsearch based application demonstrates how to perform client-side only search using itemsjs. It uses [unplatform-io/clientside-instantsearch-itemsjs-adapter](https://github.com/unplatform-io/instantsearch-itemsjs-adapter) to connect Instantsearch to itemsjs.
+<img src="https://github.com/unplatform-io/instantsearch-itemsjs-adapter/blob/main/logo-orange.svg" alt="Instantsearch-ItemsJS-adapter" align="right" width="20%" />
 
-## Getting Started
+## Introduction Instantsearch-ItemsJS-adapter 
 
-To start the project run the following commands
+This demo connects Instantsearch to a ItemsJS based search back-end, using the [Instantsearch-ItemsJS-adapter](https://github.com/unplatform-io/instantsearch-itemsjs-adapter) package. This package will perform a client side based search for your website.
+
+Instantsearch (build by Algolia) is an open-source, production-ready UI library that lets you quickly build a search interface in your front-end application. In combination with an Algolia backend, you can quickly preform searches on your website. However, it must be mentioned that Algolia is a paid service. That is one of the reasons this adapter has been build. Because of using the same front-end, users can simply upgrade from this adapter to Algolia.  
```

Comment:
I would prefer to move the focus a bit away from the cost aspect and more onto the ease-of-use aspect (plus zero additional costs for your back-end) plus the possibility to upgrade when you need more search features and/or get a larger search index (ie. more documents).